src/work/SimpleDAO.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title SimpleDAO
+ * @dev A simplified version of a DAO that allows members to create proposals,
+ */
+contract SimpleDAO {
+    struct Proposal {
+        string description;
+        bool executed;
+        uint yesVotes;
+        uint noVotes;
+        mapping(address => bool) voted;
+    }
+
+    address public owner;
+    mapping(address => bool) public members;
+    Proposal[] public proposals;
+
+    event ProposalCreated(uint proposalId, string description);
+    event Voted(uint proposalId, address voter, bool vote);
+    event ProposalExecuted(uint proposalId);
+
+    modifier onlyMember() {
+        require(members[msg.sender], "Not a member");
+        _;
+    }
+
+    constructor() {
+        owner = msg.sender;
+        // Initially, only the contract owner is a member
+        members[owner] = true;
+    }
+
+    /**
+     * @dev Allows the contract owner to add a new member.
+     * @param _member The address of the new member.
+     */
+    function addMember(address _member) public {
```

Comment:
might need a method to remove members.

Reply:
added 
    function removeMember(address _member) public {...}